packages/data-service/src/data-service.ts

```diff
@@ -1226,7 +1229,7 @@ class DataServiceImpl extends WithLogContext implements DataService {
     try {
       const cursor = this._database(databaseName, 'CRUD').listCollections(
         filter,
-        { nameOnly }
+        { nameOnly, enableUtf8Validation: false }
```

Comment:
I don't think we can do this for collection names, we would end up accessing a collection name that doesn't exist this way